### ISSUE TITLE
Icon not visible when pinned to dark taskbar in KDE Plasma

### ISSUE THREAD

**Issue by mattalexx** [rmcrackan/Libation#701]
**Describe the bug**
I have Libation pinned to my KDE Plasma taskbar, but because I am using a dark theme, the icon is invisible.

I don't have enough experience with Plasma to know if it provides developers with a solution to this issue. A kludgy solution might be to include "Use light-colored icon in taskbar" in Settings or something, but that's pretty grim.

If this bug ends up WONTFIX, I could easily probably make a light-colored version of the icon SVG and write a custom .desktop file. But what about the next guy? :-)

**To Reproduce**

1. Install Linux versions of KDE Plasma and Libation.
2. Open Plasma.
3. Open "System Settings > Appearance > Global Theme" and choose "Breeze Dark".
4. Open Libation.
5. Right click on the taskbar icon and choose "Pin to taskbar".
6. Exit Libation (to view the default state of the icon)
7. Notice the dark icon on the dark background.

**Expected behavior**

To be able to look at the pinned icon and know it would open Libation if clicked.

**Screenshots**

![Screenshot](https://i.imgur.com/em2cnYE.png)

**Platform**

```
% grep -v _URL /etc/os-release
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
LOGO=archlinux-logo
% uname -a
Linux m 6.4.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 03 Aug 2023 16:02:01 +0000 x86_64 GNU/Linux
% plasmashell --version
plasmashell 5.27.7
% kwin_x11 --version
kwin 5.27.7
% libationcli version
Libation Chardonnay v10.6.5
```

**Log Files**
n/a

**Comment by Mbucari** [comment]
Let me tell you, it's something of a minor miracle that I was even able to figure out how to do a .desktop file with icons in the first place lol.

Let me ask you, are theme-aware icons a thing? Do you know of any desktop app that can change its icon depending on the system theme?

**Comment by rmcrackan** [comment]
Damn, I thought I remembered creating this icon with a white glow for just such a reason.

**Comment by CLHatch** [comment]
> Damn, I thought I remembered creating this icon with a white glow for just such a reason.

It appears to have a sort of white outline in Windows, at least.  I've not tried a darker background yet, but my current one is somewhat dark, and I can see the white outline.
![image](https://github.com/rmcrackan/Libation/assets/13066223/0fe6afc1-dcbf-417c-a9a5-b089c87c2142)

**Comment by Mbucari** [comment]
> Damn, I thought I remembered creating this icon with a white glow for just such a reason.

Yeahhhh, I got a bit lazy when I created the sfg. Maybe I'll research adding stroke to shapes.

**Comment by mattalexx** [comment]
> Let me ask you, are theme-aware icons a thing? Do you know of any desktop app that can change its icon depending on the system theme?

I haven't ever dealt with it. I will poke around and see.

**Comment by Mbucari** [comment]
@mattalexx Try this one and let me know what you think.

https://raw.githubusercontent.com/Mbucari/Libation/master/Images/libation_glass.svg

**Comment by mattalexx** [comment]
The glow doesn't show for whatever reason. I'm looking into other options. I think the fix might need to be something like, create an icon for the Breeze theme (Plasma's default theme; light), and another for the Breeze Dark theme (Plasma's official dark theme).

Apparently, most or all third party Plasma themes "extend" one or the other of these two Breeze themes. So if I am running "Matt's Almost Perfect Custom Dark Theme" or whatever, I would reference Breeze Dark as the parent/fallback. I could include a `libation.svg` in my theme, but if I didn't, the Breeze Dark version would be used instead.

Or something like that. I haven't reached the bottom of this yet. And I'm unsure how freedesktop.org's old "hicolor" theme fits into this.

**Comment by mattalexx** [comment]
> Or something like that. I haven't reached the bottom of this yet. And I'm unsure how freedesktop.org's old "hicolor" theme fits into this.

By the way, I found that Breeze theme inherits hicolor.

**Comment by Mbucari** [comment]
@mattalexx I think github mangled the svg. Can you please try again with the copy from my repo?

https://github.com/Mbucari/Libation/blob/master/Images/libation_glass.svg

**Comment by Mbucari** [comment]
Fixed in version [v11.0.0](https://github.com/rmcrackan/Libation/releases/tag/v11.0.0)

But we may still add PNG icons later per your suggestion.
